src/sentry/monitors/logic/mark_failed.py

```diff
@@ -304,6 +326,42 @@ def create_issue_platform_occurrence(
     )
 
 
+def get_failure_reason(failed_checkins: Sequence[SimpleCheckIn]):
+    """
+    Builds a humam readible strong from a list of failed check-ins.
+
+    "3 missed check-ins detected"
+    "2 missed check-ins, 1 timeout check-in and 1 error check-in were detected"
+    "A failed check-in was detected"
+    """
+    human_names: Mapping[int, str] = dict(
```

Comment:
also nit, but do we have to declare it per function? or is it constant?

Reply:
can be constant 👍